src/main/java/com/o19s/es/ltr/feature/store/StoredFeatureSet.java

```diff
@@ -44,7 +45,7 @@
 import static org.apache.lucene.util.RamUsageEstimator.NUM_BYTES_OBJECT_HEADER;
 import static org.apache.lucene.util.RamUsageEstimator.NUM_BYTES_OBJECT_REF;
 
-public class StoredFeatureSet implements FeatureSet, Accountable, StorableElement {
+public class StoredFeatureSet implements FeatureSet, Accountable, StorableElement, ToXContentObject {
     public static final int MAX_FEATURES = 10000;
```

Comment:
StorableElement extends ToXContent; maybe it should also implement ToXContentObject?

Reply:
just did that in the last commit, also had to tweak tests